packages/dev/docs/src/docs.css

```diff
@@ -567,8 +578,8 @@ h2.sectionHeader {
     position: fixed;
     top: 0;
     left: 0;
-    width: 100vw;
-    height: 100vh;
+    width: calc(100vw - (100vw - 100%));
```

Comment:
this pattern of `100vw - (100vw - 100%)` or with `vh` makes something take 100viewport - width of scrollbar, so it won't cause the opposing scrollbar to show up because viewport sizes don't take scrollbars into account